install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -73,11 +73,11 @@ spec:
       rules:
         - alert: MCDPivotError
           expr: |
-            mcd_pivot_err > 0
+            mcd_pivot_err_total > 0
```

Comment:
Doesn't have to be part of this PR, but is this still triggerable? Pivot doesn't exist as a service anymore

Reply:
We are using this error in context of OSUpdate https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/update.go#L2217 . Pivot is the term I think we have been using since beginning, maybe we can revisit later if this name sounds confusing

Reply:
ack, sorry, I mis-read the code, I think that's fine for now then, thanks!